tests/stress/ParameterSweepTest.cpp

```diff
@@ -263,3 +271,101 @@ TEST_P(FCSweepTest, FCTest_Float16) {
   ENABLED_BACKENDS(Interpreter);
   testParamSweepFC(GetParam(), ElemKind::FloatTy, ElemKind::Float16Ty, 0.004f);
 }
+
+//===--------------------------------------------------------------------===//
+//                   Concat Parameter Sweep Tests
+//===--------------------------------------------------------------------===//
+
+/// Create a simple network that has a single fp Concat.
+static FunctionTensorPair
+createAndInitConcatNet(glow::PlaceholderBindings &bindings,
+                       glow::ExecutionEngine &EE, size_t numInputs,
+                       size_t numDims, size_t maxLength, size_t axis) {
+  PseudoRNG PRNG;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  // Make leading dimensions smaller than trailing. Reduces size of tests and is
+  // also in line with typical tests.
+  std::vector<size_t> dims(numDims, maxLength);
+  for (size_t i = 0; i < numDims; i++) {
+    dims[numDims - 1 - i] /= std::pow(2, i);
+  }
+
+  std::vector<NodeValue> inputs(numInputs);
+  for (size_t i = 0; i < numInputs; i++) {
+    auto *IP = mod.createPlaceholder(ElemKind::FloatTy, dims, "input", false);
+    bindings.allocate(IP)->getHandle().randomize(-0.2, 0.2, mod.getPRNG());
+    assert(IP);
+    inputs[i] = IP->getOutput();
+  }
+
+  auto *concat = F->createConcat("concat", inputs, axis);
+  auto *save = F->createSave("save", concat);
+  auto *resultTensor = bindings.allocate(save->getPlaceholder());
+
+  return std::make_pair(F, resultTensor);
+}
+
+/// Helper to test sweeping across a variety of configurations of a Concat by
+/// comparing the results to the Interpreter given some \p allowedError.
+/// \p config contains the backend to compare the Interpreter against, plus the
+/// specific configuration to run for this test. \p interpK and \p backendK are
+/// the element kinds to use for the Interpreter and backend, respectively.
+static void testParamSweepConcat(FourIntTupleConfig config, ElemKind interpK,
+                                 ElemKind backendK, float allowedError) {
```

Comment:
nit: s/backendK/elemKind ?

Reply:
There's `interpK` and `backendK` -- I can rename them to `interpElemKind`/`backendElemKind`.